### ISSUE TITLE
Split fields in Database

### ISSUE THREAD

**Issue by Ezekiel000** [libretro/RetroArch#13357]
I'm not sure if this is a feature request or a lack of documentation, but is it possible to have split fields in database entries?
Not sure if that's the right way to put it. For example "LEGO Star Wars" should come under both "LEGO" and "Star Wars" for franchise in explore, is this possible? Is there already a specific character that will tell retroarch to split the field?
If not I guess this is a feature request. It would be useful for Genre, Franchise, Developer, Region etc.

**Comment by RobLoach** [comment]
Yes, the metadata is available in the database: https://github.com/libretro/libretro-database

Going to move this over there.
